ipc/event/Cargo.toml

```diff
@@ -0,0 +1,7 @@
+[package]
+name = "ipc_event"
```

Comment:
I'm not sure this has to be its own crate. These events could be moved under the `ipc_provider` which emits them.